tls/s2n_kex.c

```diff
@@ -25,6 +25,14 @@
 #include "tls/s2n_tls.h"
 #include "utils/s2n_safety.h"
 
+static S2N_RESULT s2n_check_tls13(const struct s2n_cipher_suite *cipher_suite,
```

Comment:
Nit: We put the s2n_connection as the first parameter in a function usually.

Reply:
I don't choose an order, it has to match the order in the function pointer :)

Reply:
Oh ehhh why did we do that.

Reply:
Eh it's not obviously wrong. The first argument is the "module" basically, which IS usually connection. Here it would be kex though. The cipher suite has the kex, so maybe that's how it happened? We needed other fields on the cipher suite, and didn't want to add a separate arg? We just replaced kex with cipher suite?